stable/sysdig/templates/daemonset.yaml

```diff
@@ -1,8 +1,43 @@
 {{- if .Values.sysdig.AccessKey }}
```

Comment:
would you like to split up the resources (service account, clusterrole, clusterrolebindings) into different files? or all of those items in one file?